src/main/groovy/nu/studer/gradle/credentials/CredentialsPlugin.java

```diff
@@ -33,24 +32,50 @@
  */
 public class CredentialsPlugin implements Plugin<ExtensionAware> {
 
-    public static final String DEFAULT_PASSPHRASE_CREDENTIALS_FILE = "gradle.encrypted.properties";
+    public static final String DEFAULT_PASSPHRASE_CREDENTIALS_FILE = "gradle.%sencrypted.properties";
```

Comment:
nitpick:

Maybe use two fields such as 
```java
    public static final String DEFAULT_PASSPHRASE_CREDENTIALS_FILE = "gradle.encrypted.properties";
    public static final String DEFAULT_PASSPHRASE_CREDENTIALS_FILE_TEMPLATE = "gradle.%sencrypted.properties";
```
and use them respectively since;
-  this would mean changing a public field and possibly breaking consumer plugins 
- DEFAULT_PASSPHRASE_CREDENTIALS_FILE sounds like a file path or name and not a template.